CONTRIBUTING.md

```diff
@@ -33,11 +33,12 @@ make install-deps
 # install dev/tests dependencies
 make install-deps-test
 
-# code formatting (run this as a pre-commit step for your code changes)
+# code formatting
+# (this is also run automaticall as part of pre-commit hook)
```

Comment:
```suggestion
# (this is also run automatically as part of the pre-commit hook)
```

by pre-commit hook do you mean the PR test job?  If so, call it PR pre-merge testing and not pre-commit hook.

if you do mean pre-commit hook, where is this setup?

Reply:
fixed